org/TachRadio/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: org
 title: Tach Radio
-site: http://thewired.doesntexist.org
+site: http://thewireddoesntexist.org
```

Comment:
I am not sure what the again is referring here (first or second diff).
In first case (remove s in https): it is already good.
In the second case: I would like to keep the url for the organisation pointing at my personal blog.
Is it possible ?